bindings/sundials4py/arkode/arkode_lsrkstep.cpp

```diff
@@ -0,0 +1,109 @@
+/*------------------------------------------------------------------------------
+ * Programmer(s): Cody J. Balos @ LLNL
+ *------------------------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2025-2025, Lawrence Livermore National Security,
+ * University of Maryland Baltimore County, and the SUNDIALS contributors.
+ * Copyright (c) 2013-2025, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * Copyright (c) 2002-2013, Lawrence Livermore National Security.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ *----------------------------------------------------------------------------*/
+
+#include "sundials4py.hpp"
+
+#include <arkode/arkode.hpp>
+#include <arkode/arkode_lsrkstep.h>
+#include <sundials/sundials_core.hpp>
+
+#include "arkode_impl.h"
+#include "arkode_usersupplied.hpp"
+
+namespace nb = nanobind;
+using namespace sundials::experimental;
+
+namespace sundials4py {
+
+void bind_arkode_lsrkstep(nb::module_& m)
+{
+#include "arkode_lsrkstep_generated.hpp"
+
+  m.def(
+    "LSRKStepCreateSTS",
+    [](std::function<std::remove_pointer_t<ARKRhsFn>> rhs, sunrealtype t0,
+       N_Vector y0, SUNContext sunctx)
+    {
+      if (!rhs) { throw sundials4py::illegal_value("rhs was null"); }
```

Comment:
Same comments about consistently checking if rhs exists. If you chose to keep it, `None` might be preferable to `null` in message